### DISCUSSION TITLE
It says there are "outstanding issues" with my account when applying for a Students account

### DISCUSSION THREAD

**Discussion by Andrew-71** [community/community#23109] [New to GitHub]
<p>I finally decided to apply for GitHub students account today. However, when I tried sending my application I was immediately greeted with this message:</p>
<p><em>“There are outstanding issues with your GitHub account. Please contact <a href="https://support.github.com/contact" rel="noopener nofollow ugc">GitHub Support</a> for more information.”</em></p>
<p>I think I already know what it may be caused by: I live in Moscow, however I often spend part of the year in Crimea, and so my account was falsely tagged as sanctioned one some time ago. I never bothered to fix that, especially since I fear sending a picture of my passport to GitHub for privacy reasons.</p>
<p>I still believe it’s a good idea to ask here thought in case it’s some other issue. I would be extremely disappointed if the sanctioned account status is the reason for the deny, as I believe it is unfair to discriminate students based on their location, while calling the program open “to students from all countries”.</p>
<p><em>I want to state that I have provided a valid school email that matches my real life school, located in Moscow.</em></p>

**Comment by queenofcorgis** [comment]
<p>Hi there and welcome :wave:t3:</p>
<p>For student benefits and related conversations, we recommend visiting the <a href="https://github.com/github-community/community/discussions/17814">Student Developer Pack FAQs post</a> as that may help answer your questions. If not, please use that Discussion to ask for assistance as that is the Education Community’s new home.</p>

**Reply by lidi-fu** [reply]
Hi, I've also encountered the issue of "outstanding problems with my account' when I tried to apply for student pack, would you please take a look and see what's the issue here? I already verified my student email. Thank you so viewing!

**Reply by micZ3** [reply]
I am glad to tell you that I have successfully solved this problem.You can contact github, email them and tell them that there is something wrong with your account, and then it will be solved.Then check whether your billing information is the same as your name, and then wait for the result.

**Comment by Andrew-71** [comment]
<p>Hello, thanks for your reply! :smiley:</p>
<p>You seem to have accidentally sent the wrong link, however I have found what I presume to be <a href="https://github.community/t/github-student-developer-pack-faq/124041">the correct page</a> and read it throughout, plus re-read the <a href="https://education.github.com/discount_requests/student_application" rel="noopener nofollow ugc">registration page requirements</a>. It seems I qualify to all points:</p>
<ol>
<li>“Be currently enrolled in a degree or diploma granting course of study” - Yes, I am currently in school, having finished grade 9 today.</li>
<li>“Have a verifiable school-issued email address” - Yes, I have provided an email address with my school’s personal and easily verifiable domain.</li>
<li>“Have a GitHub user account.” - Obviously, I do! :slight_smile:
</li>
<li>“Be at least 13 years old.” - I am currently 15 years old.</li>
</ol>
<p>Therefore I believe the only reason it could automatically not let me apply could be, in fact, my account being tagged as a Crimean one. I have even given the website my location data which proves I am in same city as my school - Moscow, which is obviously not in a sanctioned region.</p>
<p>I would like to know if I’ll have to send out a blurred out pic of my passport with “Issued by Moscow gov” on it to qualify for students program or if there is an easier way to do it.</p>

**Comment by queenofcorgis** [comment, marked as answer]
<p>Thank you for letting me know about the link, I’ve updated it :smile:</p>
<p>But I do understand that is frustrating and I wish I could help further besides pointing in the right direction.</p>
<p>I would suggest posting your topic there in the Education category as those folks are going to be more likely to be able to help you than the folks who work on this Forum.</p>

**Comment by Andrew-71** [comment]
<p>Alright, thanks! I’ll post it there. If I won’t find the solution I’ll just try getting the sanctioned badge off my account :smile:</p>

**Reply by ChestnutLord** [reply]
Привет. А как ты понял, что твой аккаунт попал под санкции? В нём появился какой-то лейбл? У меня такая же проблема, но визуально с аккаунтом всё норм.

**Comment by RabieTAWIL** [comment]
Dear @Andrew-71 ,
I hope you are doing well. I recently wanted also to participate in the Student Developer Pack, and got the same error that you had. Similarly, I think my problem is similar to yours where my account were once accessed from a region that is under some restriction, and I didn't do anything about it that time, but I wish I did.

I would like to ask if you solved your problem, and if you can help and guide me to solve mine in case you solved yours.

Thank you in advance.

**Reply by Andrew-71** [reply]
Hey! I did end up getting students discount, by going trough the sanction appeal process. If you have sanctioned account you can see a button to start one on most warnings about it (e.g when creating a repository there should be one near private repository option). You’ll need a passport from outside the sanctioned region and a selfie thought. Hope this helps!

**Reply by RabieTAWIL** [reply]
Thanks a lot for your help. I did your instruction for now and hope it will resolve my issue.

Best regards :)

**Reply by FancyangY** [reply]
Hi,,I got the same error that you had.but i don't have the option to cancel got the same error that you had

**Reply by micZ3** [reply]
how to do it? I'm so sad

**Reply by HorselessName** [reply]
This is really annoying, I am also sad. Again the world is so unfair for lower social classes. if i was middle-high class this would be solved in 5 minutes

**Comment by HorselessName** [comment]
I am having the same problem

**Reply by loveyouandmyself** [reply]
I am having the same problem

**Comment by jgc9527** [comment]
I have the same problem.

**Comment by queenofcorgis** [comment]
Hi there folks - this is an older discussion so the relevant folks may not see these comments, if you have a question or situation with the student developer pack I recommend opening a new discussion in the [Education Category](https://github.com/orgs/community/discussions/categories/github-education?discussions_q=category%3A%22GitHub+Education%22+) if you've searched and don't see a solution. cc: @Miu-Lee
